docs/dev/table/functions.md

```diff
@@ -2449,6 +2449,30 @@ REPEAT(string, integer)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+ASCII(string)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns a numeric value of the leftmost character of the <i>string</i>.</p> 
```

Comment:
Returns the ASCII code value of... (please also update the Javadoc correspondingly)

As we've reached an agreement on adding the return type, it's better to mention it here.